meta-balena-reference/layers/meta-balena-bsp/recipes-support/hostapp-update-hooks/hostapp-update-hooks.bbappend

```diff
@@ -0,0 +1 @@
+HOSTAPP_HOOKS += "99-resin-grub"
```

Comment:
Is 99-resin-grub qemu specific, or is it grub specific? Should it move to meta-balena?